discovery-provider/src/api/v1/users.py

```diff
@@ -914,7 +927,7 @@ def _get(self, id):
         responses={200: "Success", 400: "Bad request", 500: "Server error"},
     )
     @full_ns.expect(user_favorited_tracks_parser)
-    @full_ns.marshal_with(favorites_full_response)
```

Comment:
@nicoback2 this also changes the response type for the `USER_FAVORITED_TRACKS_ROUTE` endpoint (`"/<string:id>/favorites/tracks"`) - it's a more precise type but not sure if this will create more trouble than it's worth? LMK

Reply:
@piazzatron is it a breaking change for client? if so would rather leave this change out, otherwise will be fine considering the old saga code for favorites is JS lol